tests/roles/swift_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,91 @@
+- name: Add swift.conf secret
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    CONTROLLER1_SSH="{{ controller1_ssh }}"
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: swift-conf
+      namespace: openstack
+    type: Opaque
+    data:
+      swift.conf: $($CONTROLLER1_SSH cat /var/lib/config-data/puppet-generated/swift/etc/swift/swift.conf | base64 -w0)
+    EOF
+
+- name: Add swift ring files configmap
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    CONTROLLER1_SSH="{{ controller1_ssh }}"
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: swift-ring-files
+    binaryData:
+      swiftrings.tar.gz: $($CONTROLLER1_SSH "cd /var/lib/config-data/puppet-generated/swift/etc/swift && tar cz *.builder *.ring.gz backups/ | base64 -w0")
+    EOF
+
+- name: Deploy podified Swift
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      swift:
+        enabled: true
+        template:
+          secret: osp-secret
+          swiftConfSecret: swift-conf
+          memcachedInstance: memcached
+          swiftRing:
+            ringReplicas: 1
+          swiftStorage:
+            replicas: 0
+            networkAttachments:
+            - storage
+            storageClass: local-storage
+            storageRequest: 10Gi
+          swiftProxy:
+            secret: osp-secret
+            replicas: 1
+            passwordSelectors:
+              service: SwiftPassword
+            serviceUser: swift
+            override:
+              service:
+                internal:
+                  metadata:
+                    annotations:
+                      metallb.universe.tf/address-pool: internalapi
+                      metallb.universe.tf/allow-shared-ip: internalapi
+                      metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                  spec:
+                    type: LoadBalancer
+            networkAttachments:
+            - storage
+    '
+
+- name: Wait for Swift to start up
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get pod -l app.kubernetes.io/name=SwiftProxy -o jsonpath='{.items[0].status.phase}{"\n"}' | grep Running
```

Comment:
This needs to be changed once https://github.com/openstack-k8s-operators/swift-operator/pull/126 is merged and the openstack-operator bumped the swift-operator version. Should be "oc get pods -l component=swift-proxy" then

Reply:
Done